x-pack/plugins/uptime/server/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -18,27 +18,28 @@ export const syntheticsMonitor: SavedObjectsType = {
     dynamic: false,
     properties: {
       name: {
-        type: 'keyword',
+        type: 'text',
+        fielddata: true,
       },
       id: {
         type: 'keyword',
       },
       type: {
-        type: 'keyword',
+        type: 'text',
       },
       urls: {
-        type: 'keyword',
+        type: 'text',
       },
       tags: {
-        type: 'keyword',
+        type: 'text',
       },
       secrets: {
```

Comment:
This is my fault, but can we remove this. There is no reason to have secrets indexed.